uc-crux-llvm/src/UCCrux/LLVM/Main/Config/FromEnv.hs

```diff
@@ -1,14 +1,20 @@
 {-
-Module       : UCCrux.LLVM.Config
-Description  : CLI, environment, and config file options
+Module       : UCCrux.LLVM.Config.FromEnv
+Description  : Grab configuration from CLI, environment variables, and config file
 Copyright    : (c) Galois, Inc 2021
 License      : BSD3
 Maintainer   : Langston Barrett <langston@galois.com>
 Stability    : provisional
+
+The functions/types in this module aren't necessarily appropriate for using
+UC-Crux-LLVM as a library: Some of them are impure, and they can throw
+exceptions. Moreover, 'UCCruxLLVMOptions' is a monolithic datatype that combines
+configuration options for a wide variety of functionality, which is probably
+unnecessary for most library use-cases.
```

Comment:
> It might help to expand on it to confirm my hypothesis that these are all "top-level" functions used by library clients for integration into the larger crucible toolset and that they are not required for using the uc-crux-llvm library in other configurations

I'm not sure I understand your hypothesis here, can you expand on this?

> the functions here are not called internally by the library itself.

I believe this is true and I can add a comment to that effect.

Reply:
The hypothesis is essentially confirmed by your agreement with the latter: that these functions are convenience for crucible integration and neither required by nor called by the rest of the library.

Reply:
Comment added, thanks!